src/components/Calendar.tsx

```diff
@@ -153,7 +160,7 @@ export const Calendar = ({ query, queryVariables, calendarData, isLoading, navig
 
       {subList && (
         <ListComponent
-          data={buildListItems(data)}
+          data={loading ? [] : buildListItems(data)}
           horizontal={false}
```

Comment:
I found out, that `horizontal` is not needed here, or am I wrong?

Reply:
same for `showBackToTop`, which does not exist for `EventList`

Reply:
Yes, you are right! https://github.com/smart-village-solutions/smart-village-app-app/pull/855/commits/2a84d5989617f2fd9b25e76f484c2352ccab602c